lit.ts

```diff
@@ -478,6 +491,173 @@ export async function getPubkeyForAuthMethod({
 	return pubkey;
 }
 
+export async function sendLitTokens(recipientPublicKey: string, amount: string) {
+	const signer = getSigner();
+
+	const tx = await signer.sendTransaction({
+		to: recipientPublicKey,
+		value: parseEther(amount),
+	});
+
+	const reciept = await tx.wait();
+
+	console.log("Sent LIT tokens", reciept.blockHash);
+
+	return reciept.blockHash;
+}
+
+export async function mintCapacityCredits({
+	signer,
+}: {
+	signer: ethers.Wallet;
+}) {
+	if (config.network !== "habanero" && config.network !== "manzano") {
```

Comment:
`Manzano` does support `CapacityDelegation` and if we send requests with it it wont effect validation if the network is not configured for payment.

Reply:
The purpose of this check is due to the "getContractFromWorker" function below. It will only accept habanero or manzano.